tests/test_two_sided_market.py

```diff
@@ -391,56 +431,16 @@ def test_matching_list_gets_updated_with_residual_offers(self):
         assert matches[1]["bid"]["id"] == "residual_bid_2"
 
 
-@pytest.fixture()
-def two_sided_market_matching():
-    patches = [
-        patch("gsy_e.models.market.two_sided.TwoSidedMarket.validate_bid_offer_match"),
-        patch("gsy_e.models.market.two_sided.TwoSidedMarket.accept_bid_offer_pair"),
-        patch("gsy_e.models.market.two_sided.TwoSidedMarket."
-              "_replace_offers_bids_with_residual_in_recommendations_list"),
-    ]
-    for p in patches:
-        p.start()
-    yield
-    for p in patches:
-        p.stop()
-
-
 class TestTwoSidedMarketMatchRecommendations:
     """Class Responsible for testing two sided market's matching functionality."""
-    def test_match_recommendations_fake_offer_bid(self, market, two_sided_market_matching):
```

Comment:
Why was this removed?

Reply:
I have moved the validation of existence of orders from the `match_recommendations` to the `validate_bid_offer_match` method, thus this test was removed and another test was added, please have a look at the `test_validate_bid_offer_match_orders_do_not_exist`

Reply:
I see, thanks and sorry for overseeing!